src/index.html

```diff
@@ -11,7 +11,119 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="group__field">
+        <legend>Personal information</legend>
+        <label class="form__field">Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          >
+        </label>
+        <label class="form__field">Name:
+          <input
+          type="text"
+          name="name"
+          autocomplete="off"
+          >
+        </label>
+        <label class="form__field">How old are You?
+          <input
+          type="number"
+          name="Age"
+          min="1"
+          max="100"
+          value="12"
+          >
+        </label>
+        <label class="form__field">Full date of birth:
+          <input
+          type="date"
+          name="date"
+          >
+        </label>
+        <label class="form__field">I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="remember"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group__field">
+        <legend>Registration</legend>
+        <label class="form__field">E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          required
+          >
+        </label>
+        <label class="form__field">Password:
+          <input
+          type="password"
+          name="password"
+          minlength="5"
+          maxlength="20"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group__field">
+        <legend>An interesting fact about you!</legend>
+        <label class="form__field">Do you love cats?
+          <input
+          type="radio"
+          name="Yes"
+          >Yes
+          <input
+          type="radio"
+          name="No"
+          >No
+        </label>
```

Comment:
wrap each input in label and just inputs.
now click on `do you love cats ` select for radio input
something like that
```suggestion
        <div class="form__field">Do you love cats?
          <label>
            <input
              type="radio"
              name="Yes"
            >
            Yes
            </label>
            <label>
            <input
              type="radio"
              name="No"
            >
            No
           </label>
        </div>
```
now possible to select to radio inputs
for fix it set same name for each input in radio inputs group
![Screenshot from 2023-04-07 12-02-41](https://user-images.githubusercontent.com/33029864/230579912-e5950470-be87-4675-9562-117304bcbbf1.png)

Reply:
I did such change, it's work good